Makefile

```diff
@@ -41,11 +38,15 @@ install:
 	yarn --pure-lockfile
 
 .PHONY: dev 
-dev: deps
-	deps/hugo serve --theme hugo-whisper-theme
+dev: 
+	# If $(DEPS) are available via `make deps` use those executables. Otherwise, rely on executables to already be installed. 
+	$(DIR)hugo serve --theme hugo-whisper-theme
 
 .PHONY: test
-test: deps deps-proofer
+test: 
 	rm -rf public
-	deps/hugo -v -s .
-	htmlproofer --allow-hash-href --check-html --empty-alt-ignore --url-ignore /kots.io/css/ "./public"
\ No newline at end of file
+	# Use hugo dependencies via `make deps`, if available. Otherwise use OS install of Hugo.
+	$(DIR)hugo -v -s .
```

Comment:
what do you think about pulling `$(DIR)hugo` into ` `$(HUGO)` var?

Reply:
then it becomes a make var that folks can override to a specific path if they want

Reply:
Good idea.

Reply:
OK, updated accordingly.